metrics/prometheus-metrics-endpoints-and-auto-discovery.md

```diff
@@ -0,0 +1,136 @@
+# Abstract
+
+This document outlines a proposal for gathering metrics from Mobile Services running in a Kubernetes environment and visualizing them.
+
+## Goals
+
+### Primary Goals
+
+* Provide a mechanism for Mobile Developers to gather, view & query metrics from supported Mobile Services.
```

Comment:
I might be getting this wrong but saying "mobile developers" is misleading and my suggest mobile metrics. 
I will suggest to rename mobile to "DevOps" or "Server side" developers. Considering non goals bellow there is no specific mobile focus and it can be simply added as "openshift metrics service"

Reply:
Good point. I can rephrase.
How does this sound for the primary

"Provide a mechanism for developers to gather, view & query metrics from supported Mobile Services"

and the secondary

"Allow Developers to leverage the same mechanism for any other/non-mobile/custom services"